changelog.d/14020.misc

```diff
@@ -0,0 +1 @@
+Clear out stale entries in `event_push_actions_staging` table.
```

Comment:
We have code meant to handle this: https://github.com/matrix-org/synapse/blob/8625ad80994d6049a778b5d1ef65c8d1b1042c74/synapse/storage/databases/main/event_push_actions.py#L825-L842

I'm guessing these entries could still be here from things like restarts? How else could they be here?

Reply:
Yeah, restarts I think should be the main cause of them, or there's been a couple of places we forgot to call `remove_push_actions_from_staging` from previously?

Reply:
We should investigate it if we continue seeing a build up of rows in the staging table after we've purged the current stale entries.

Reply:
OK! Should we file an issue about this?

Reply:
done: https://github.com/matrix-org/synapse/issues/14029